src/script/index.js

```diff
@@ -12,6 +15,20 @@ const toggleHamburger = () => {
     authWrapper.classList.toggle('active');
 };
 
+/**
+ * Function to check for valid email
+ */
+const checkEmail = (e) => {
+    e.preventDefault();
+    const email = emailInput.value;
+
+    if (email == '' || !emailRegex.test(email)) {
+        emailWrapper.classList.add('input__text--error');
```

Comment:
just a red outline is not enough. Also add a very light red background color
![image](https://github.com/user-attachments/assets/746147e5-1c10-41da-b272-86134335f52e)